sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- Report Performance Counters "CPU Total" and "Committed Memory".
```

Comment:
do we want to use these names in changelog?

Reply:
Don't even call it as performance counters.